packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -630,6 +645,8 @@ test('price drop', async t => {
     shortfallBalance: AmountMath.makeEmpty(runBrand),
   });
   await m.assertChange({
+    // FIXME expects to know liquidity brand
```

Comment:
these tests check the brand, but they don't know the brand of the liquidity. maybe a mock will work